.github/workflows/delete_backport_branch.yml

```diff
@@ -0,0 +1,15 @@
+name: Delete merged branch of the backport PRs
+on: 
+  pull_request:
+    types:
+      - closed
+  
+jobs:
+  delete-branch:
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.pull_request.head.ref,'backport/')
+    steps:
+      - name: Delete merged branch
+        uses: SvanBoxel/delete-merged-branch@main
```

Comment:
We should not use branch references for security reasons, this should be a commit id or tag to be sure only the approved version of the plugin is running